src/index.js

```diff
@@ -7,9 +7,16 @@ import {ValidationReporter} from 'aurelia-validation';
 import {ValidationReporter as ValidateJSReporter} from './validation-reporter';
 export {ValidationReporter} from './validation-reporter';
 export {ValidationRenderer} from './validation-renderer';
+import {ValidationConfig} from './validation-config';
+import {ConfigBuilder} from './config-builder';
 
-export function configure(config) {
-  config.container.registerHandler(Validator, ValidateJSValidator);
-  config.container.registerHandler(ValidationReporter, ValidateJSReporter);
-  config.globalResources('./validate-binding-behavior');
+export function configure(aurelia, config) {
+  aurelia.container.registerHandler(Validator, ValidateJSValidator);
+  aurelia.container.registerHandler(ValidationReporter, ValidateJSReporter);
+  aurelia.globalResources('./validate-binding-behavior');
+
+  ValidationConfig.prototype.configBuilder = new ConfigBuilder();
```

Comment:
setting things on the prototype isn't the nicest solution, but we need to get this ConfigBuilder instance from `validation-config.js` that doesn't use DI. Is it OK to use `Container.instance.get()` there?